libraries/chain/block.cpp

```diff
@@ -133,11 +133,11 @@ namespace eosio { namespace chain {
       return validate_and_extract_block_extensions( block_extensions );
    }
 
-   signed_block_v0_ptr signed_block::to_signed_block_v0() const {
+   signed_block_v0_uptr signed_block::to_signed_block_v0() const {
```

Comment:
I prefer to return optional<signed_block_v0> based on how it is used.

Reply:
Changed